design/authenticators-status/authn-status-new-implem.md

```diff
@@ -0,0 +1,132 @@
+# Implementing Authenticator Status API
+The authenticator status API allows operators to get immediate feedback on the authenticator configuration. Once the configuration of an authenticator is complete, the operator sends a status request and the status of the given authenticator is returned. 
+
+If the configuration was successful, the response received will reflect this. If not, an error will be provided in the response, identifying the step of failure.
+
+The first authenticator to utilize this feature is Authn OpenID Connect. For more information about the design behind the Status API and implementation details see the [Status epic](https://github.com/cyberark/conjur/issues/1062). 
+It is advised that you read the epic before expanding status to other authenticators.
+
+_For a complete status flow [diagram](https://github.com/cyberark/conjur/blob/master/design/authenticators-status/authn-status-flow.jpeg)_
+
+This doc describes how to expand the status functionality to other authenticators.
+For each authenticator, a suite of [general](https://github.com/cyberark/conjur/blob/master/design/authenticators-status/authn_status_general.md) 
+checks are run followed by those specific to the authenticator.  
+
+General checks are outlined below and steps that require developer intervention to implement and enable status for authenticators are italicized accordingly.
+1. The requesting user has access to the authenticator status webservice
+    1. See the security section [here](https://github.com/cyberark/conjur/issues/1062) for more info about this webservice 
+1. _The authenticator implements the status method_
+1. The account exists
+1. The webservice exists
+1. The authenticator is enabled in the ENV
+1. _Checks specific requirements for the new authenticator_
+
+To properly implement the status functionality in the new authenticator, you will need to expand on italicized checks 2 and 6. 
+As described in step 2, the status of an authenticator method is checked for implementation. Therefore, you will need to add a status method to the Authenticator class (`Authentication::Authn<Type>::Authenticator`) like so:
+
+```ruby
+def status(authenticator_status_input:)
+  Authentication::Authn<Type>::ValidateStatus.new.(
```

Comment:
can you change this to `Authentication::Authn<Type>::ValidateStatus.new.call(`?

Reply:
Also, i think the three dots don't help much. i think that `<ValidateStatus input>` is a bit better